public/app/core/components/AppChrome/AppChromeService.tsx

```diff
@@ -24,8 +24,8 @@ export interface AppChromeState {
   kioskMode: KioskMode | null;
   layout: PageLayoutType;
   returnToPrevious?: {
-    href: ReturnToPreviousProps['href'];
     title: ReturnToPreviousProps['title'];
+    href?: ReturnToPreviousProps['href'];
```

Comment:
this should also be mandatory. by the time it's set in state, it must always be defined (if title is defined)